Automattic/wp-calypso
# Checkout: Fetch PayPal client ID from configuration endpoint

## Proposed Changes

This PR modifies the in-development PayPal PPCP payment method (see https://github.com/Automattic/wp-calypso/pull/94790) so that it fetches its PayPal client ID from a new configuration endpoint rather that being hard-coded. This will allow us to use different client IDs for test and production and to change the ID more easily.

We create a new npm package, `@automattic/calypso-paypal` to handle this fetching and store the ID in context so it can be reused more easily in the future. This mirrors the approach we use for Stripe and Razorpay, which each have their own packages also.

Requires D167324-code

## Testing Instructions

- Visit checkout with a product in your cart.
- Verify that the PayPal (PPCP) payment method is one of the options.
- Verify that when submitting the transaction, the correct client ID is used.
